src/libAtomVM/nifs.gperf

```diff
@@ -119,6 +119,7 @@ erts_debug:flat_size/1, &flat_size_nif
 atomvm:add_avm_pack_binary/2, &atomvm_add_avm_pack_binary_nif
 atomvm:add_avm_pack_file/2, &atomvm_add_avm_pack_file_nif
 atomvm:close_avm_pack/2, &atomvm_close_avm_pack_nif
+atomvm:get_entry_point/1, &atomvm_get_entry_point_nif
```

Comment:
I'm not sure about this API:
1. maybe this can be changed to something more generic, such as `find_section_by_flag` or `avmpack_find_entry_by_flag`. However I think at the moment we just support `BEAM_START_FLAG` so it might be an overkill having something too generic.
2. maybe entry_point is not a good name and it should be changed to get_start_beam or even existing naming might be changed to entry point.

Reply:
I think get_start_beam might be the most clear.